week2/homework/src/index.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+let fs = require ('fs');
```

Comment:
Variable `fs` is not re-assigned, so it has to be declared with `const`

Reply:
thanks for the advice..